apps/web/next.config.js

```diff
@@ -175,7 +175,7 @@ const nextConfig = {
   output: process.env.BUILD_STANDALONE === "true" ? "standalone" : undefined,
   experimental: {
     // externalize server-side node_modules with size > 1mb, to improve dev mode performance/RAM usage
-    serverComponentsExternalPackages: ["next-i18next"],
+    serverComponentsExternalPackages: ["next-i18next", "typeorm", "@boxyhq/saml-jackson", "handlebars"],
```

Comment:
saml-jackson imports a bunch of things nextjs will try  to bundle by default. We don't want them to as theyre pretty big so we just let node take care of it.